.github/workflows/update-chatbot.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     name: Update Chatbot (Preview)
     runs-on: ubuntu-latest
     environment: preview
-    if: github.ref == 'refs/heads/preview'
+    if: github.repository == 'alveusgg/alveusgg' && github.ref == 'refs/heads/preview'
```

Comment:
I think the preview we still want to be able to deploy from the private repo if needed, just not production as the public repo main branch handles that.